emf/metamodel.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I think this one should not be committed (it is probably my fault that this file is generated here by some test)

Reply:
I updated the `emf/.gitignore` file and removed `metamodel.json`